api/internal/handler/nft_handler.go

```diff
@@ -54,7 +64,7 @@ func NewUpdateNftBackstoryHandler(nftTokenRepository repository.NftTokenReposito
 			return
 		}
 
-		if nftToken.OwnerAddress != requestBody.WalletAddress {
+		if *nftToken.OwnerAddress != requestBody.WalletAddress {
```

Comment:
do you need to check if nftToken is null?

Reply:
The pointer is on the OwnerAddress. Updated